sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 1.4.0b3 (Unreleased)
 
+- Preview of `VSCodeCredential` #10472
+
+VS code credential is to authenticate with the identity signed in to Visual 
+Studio Code's Azure extension. This credential is internal because it is 
+exposed to users only as part of an aggregate credential. 
```

Comment:
From a user's perspective, authenticating by signing in to VS Code is a feature of `DefaultAzureCredential`. Mentioning `VSCodeCredential` and describing its implementation is a red herring because it's an internal class we don't want anyone to import.